src/App.tsx

```diff
@@ -160,20 +161,24 @@ const App = () => {
 
     return (
         <div id={"app"}>
-            <ThemeContextProvider>
-                <LocalizationProvider dateAdapter={AdapterDayjs}>
-                    <DropFile onFileDrop={handleFileChange}>
-                        {(APP_STATE.FILE_VIEW === appMode) &&
-                        <Viewer
-                            enablePrettify={enablePrettify}
-                            fileSrc={fileSrc}
-                            initialQuery={initialQuery}
-                            logEventNumber={logEventIdx}
-                            seekParams={initialSeekRef.current}
-                            timestamp={timestamp}/>}
-                    </DropFile>
-                </LocalizationProvider>
-            </ThemeContextProvider>
+            <LocalizationProvider dateAdapter={AdapterDayjs}>
+                <CssVarsProvider>
+                    <ThemeContextProvider>
+                        <DropFile onFileDrop={handleFileChange}>
+                            {(APP_STATE.FILE_VIEW === appMode) &&
+                            <Viewer
+                                enablePrettify={enablePrettify}
+                                fileSrc={fileSrc}
+                                initialQuery={initialQuery}
+                                logEventNumber={logEventIdx}
+                                seekParams={initialSeekRef.current}
+                                timestamp={timestamp}
+                                timezone={"local"}/>}
```

Comment:
Since we're not reading this "initial" `timezone` value from the URL at the moment, we don't have to add the `timezone` prop here.